workers/ohsome_quality_analyst/api/api.py

```diff
@@ -223,7 +238,17 @@ async def _fetch_report(
         fid_field=fid_field,
         size_restriction=True,
     )
+
     response = empty_api_response()
+    if includeSvg is False:
+        cnt = 0
+        while True:
```

Comment:
Here I recommend to use a for loop which will for sure terminate.

Also instead of building the expected string, pattern matching could be used here.

```
import fnmatch

pattern = "*.result.svg"
string = "indicators.1.result.svg"

fnmatch.fnmatch(string, pattern) # True
```